jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/sam/SamExecutable.kt

```diff
@@ -305,7 +304,6 @@ fun GeneralCommandLine.samSyncCommand(
         addParameter("--use-container")
     }
     addParameter("--no-dependency-layer")
-    if (syncOnlyCode) {
-        addParameter("--code")
-    }
+
+    addParameter("--no-watch")
```

Comment:
do we need to worry about older SAM CLI versions that can't handle this flag?

Reply:
This new change requires an upgrade to the latest version that supports this change aka 1.78.0